lib/utils/ember.js

```diff
@@ -298,16 +298,55 @@ function isObserverProp(node) {
   return isPropOfType(node, 'observer');
 }
 
-function isComputedProp(node) {
+function isComputedProp(
+  node,
+  importedEmberName,
+  importedComputedName,
+  { includeSuffix, includeMacro } = {}
```

Comment:
Could you add comments explaining these arguments?

Reply:
Good idea, added jsdoc comments.